modules/tensor_mechanics/doc/content/documentation/systems/Modules/TensorMechanics/GeneralizedPlaneStrain/index.md

```diff
@@ -7,7 +7,7 @@
 This action sets up a generalized plane strain model. A detailed description of generalized plane strain model can be found in the [formulation](tensor_mechanics/generalized_plane_strain.md) page.
 
 !alert warning title=For 2D and 1D Simulations
-GeneralizedPlaneStrain only works for 1D axisymmetric or 2D generalized plane strain cases where the number of displacement variables is only one or two displacements.
+GeneralizedPlaneStrain only works for 1D axisymmetric or 2D generalized plane strain cases. For 1D axisymmetric and 2D generalized plane strain cases in the x-y plane the number of displacement variables must be one or two displacements, respectively. For 2D generalized plane strain cases in the x-z or y-z planes the number of displacement variables must be three.
```

Comment:
Make the last sentence a separate paragraph (you can keep it in the same alert by using `!alert!` and `alert-end!` [Docs](http://www.mooseframework.org/moose/utilities/MooseDocs/extensions/alert.html) ).

Bold the planes so that the difference is clear: `+x-z or y-z planes+` and `+x-y plane+`